src/content/docs/blog/2025/09/30/September 2025 project update/index.mdx

```diff
@@ -0,0 +1,69 @@
+
+Whilst AerynOS is a rolling release distribution, we don't aim to be on the bleeding edge just for the sake of being on the bleeding edge.
+
+## Gnome 49
+
+The team updated to Gnome 49 in the fortnight following its release meaning it's now available in our repository. Given the way Gnome updates and how extensions then follow suit, we are aware that many extensions have not yet been updated for compatibility with Gnome 49.
+
+As such, we are holding back our next live installer ISO release until a larger number of extensions have had time to go through their own update cycles.
+
+## Infra stabilization workstream
+
+2025 has seen a massive lift in our infra code with the re-write to Rust taking place in Q2, and its stabilisation being pursued from the end of Q2 and up until now.
+
+In this period, the team has been incrementally refining the code, and resolved bugs as they have arisen. This is work that average users will not have seen or been aware of as it only affects workflows behind the scenes. However, this has been extremely important work that has increasingly allowed us to become more confident in our new infra foundations. 
+
+We are now at the point where we are confident in calling our new infrastructure "stable", with no bugs having presented themselves in several weeks and hundreds of packages built. 
+
+It cannot be overstated how much better the state of our infra is compared to the start of the year. In turn, this now allows us to focus our efforts in other areas of our technology stack to similarly work through bugs, implement features and generally refine our code base.
+
+## Cosmic maintainership
+
+During the last couple of months, we had a new contributor join us and begin working on the Cosmic stack. His contributions have been invaluable to the team and we thank him for his engagement. 
+
+Unfortunately, due to personal reasons, he is having to take a temporary step back from the project. He has shared his methodology for providing updates to the Cosmic packages and we already have willing contributors picking up the tasks from which he was forced to step back.
+
+We know he will be back once his circumstances improve, but in the meantime, if you are curious about supporting Cosmic on AerynOS, please do get in touch in our [Matrix packaging channel](https://matrix.to/#/#AerynOS-Packaging:matrix.org) as it's always good to have several people capable of maintaining each of our DE stacks.
+
+## Web development
+
+One area in which we often receive feedback is our website. Whilst we are in our Alpha state, it has not been our highest priority to "fix". We do have a couple of interested contributors wanting to work on our website to improve the overall look and feel of the project, though.
```

Comment:
I think the `"fix"` needs some elaboration about why it's broken in the first place. Or maybe just replace by 'improve'?